src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/OffenderProgramProfile.kt

```diff
@@ -89,6 +89,9 @@ data class OffenderProgramProfile(
   @Column(name = "SUSPENDED_FLAG")
   @Convert(converter = YesNoConverter::class)
   var suspended: Boolean = false,
+
+  @OneToMany(mappedBy = "offenderProgramProfile", cascade = [CascadeType.ALL], fetch = FetchType.LAZY)
+  val offenderCourseAttendances: MutableList<OffenderCourseAttendance> = mutableListOf(),
```

Comment:
Had to add the attendances as a child of the allocation as this is how we model the data in the DSL.